lz4/PKGBUILD

```diff
@@ -0,0 +1,127 @@
+# Maintainer: Sébastien Luttringer
+# PGO build: Laio O. Seman <laio [at] ieee.org>
+
+pkgname=lz4
+epoch=1
+pkgver=1.9.4
+pkgrel=2
+pkgdesc='Extremely fast compression algorithm'
+arch=('x86_64')
+url='https://lz4.github.io/lz4/'
+license=('GPL2')
+makedepends=('git' 'cmake' 'ninja')
+checkdepends=('diffutils')
+depends=('glibc')
+source=("git+https://github.com/lz4/lz4.git#tag=v$pkgver")
+sha256sums=('SKIP')
+
+# Function to generate data and compress-decompress it
+generate_and_process_data() {
```

Comment:
The function should be local.

Reply:
I meant to just add underscore before the function name like arch package guidelines says. It is not necessary to use the local keyword here.